### PR TITLE
:sparkles: New tvl pools

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206536112877301

### CODE REVIEW

File: views/metrics/components/chart-wrapper/index.tsx [hunk 1]
```diff
@@ -10,9 +10,9 @@ const ChartWrapper: FC<PropsWithChildren<ChartHeaderProps>> = ({
 }) => (
   <Box
     width="100%"
+    height="100%"
     display="flex"
     borderRadius="xs"
-    overflowY="hidden"
     color="onSurface"
     gridColumn="1/-1"
```
Comment (MarioBatalha) [inline views/metrics/components/chart-wrapper/index.tsx:17]: If direct parent is not display grid, please move it cuz there is not need for that. GridColumn hasn't effect for display flex.

File: views/metrics/index.tsx [hunk 2]
```diff
@@ -20,6 +21,16 @@ const Metrics: FC = () => (
     </Typography>
     <Box variant="container">
       <DailyVolume />
+      <Box
```
Comment (MarioBatalha) [inline views/metrics/index.tsx:24]: Change the Box to Typography and use an array of font size.

File: views/metrics/tvl-pools/index.tsx [hunk 3]
```diff
@@ -0,0 +1,107 @@
+import { Box, Chart } from '@interest-protocol/ui-kit';
```
Comment (MarioBatalha) [inline views/metrics/tvl-pools/index.tsx:1]: In general, the component is not completely responsive mainly the tabs

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

File: views/metrics/components/chart-wrapper/index.tsx (hunk 1 above)
Comment (KipandaJr) [reply views/metrics/components/chart-wrapper/index.tsx:17]: We give u more details about this on daily

File: views/metrics/tvl-pools/index.tsx [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+          <Box className="box-pie">
+            <Chart
+              variant="pie"
+              label="Pools"
+              data={data.map((item) => ({
+                date: item.label,
+                amount: item.amount,
+                description: item.label,
+              }))}
+              semanticColors={[
```
Comment (git-marcopitra) [inline views/metrics/tvl-pools/index.tsx:88]: Move it to a `const` outside this file

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Pull request merged**